src/constants.php

```diff
@@ -7,4 +7,4 @@
 // independent of selfoss version
 // needs to be bumped each time public API is changed (follows semver)
 // keep in sync with docs/api-description.json
-const SELFOSS_API_VERSION = '4.0.0';
+const SELFOSS_API_VERSION = '5.0.0';
```

Comment:
Is the api version 4.1.0 or 5.0.0?

Reply:
`5.0.0` since the introduction of `403 Forbidden` errors is technically a breaking change.